Library/Formula/git-town.rb

```diff
@@ -0,0 +1,40 @@
+class GitTown < Formula
+  desc "High-level command-line interface for Git"
+  homepage "http://www.git-town.com"
+  url "https://github.com/Originate/git-town/archive/v0.7.3.tar.gz"
+  sha256 "947fd6dac603dcb02467cc2969ef26db1ccdc72312b6b0d70fb8d11179eb09ba"
+
+  depends_on "dialog" => :recommended
+
+  def install
+    # Install the source
+    libexec.install Dir["src/*"]
+
+    # Symlink the executables
+    bin.install_symlink Dir["#{libexec}/git-*"]
+    bin.install_symlink "#{libexec}/helpers"
+    bin.install_symlink "#{libexec}/drivers"
```

Comment:
Instead of dumping these all in the `bin`, if you do this instead it should solve the audit warning and leave the executables available:


```ruby
bin.write_exec_script Dir["#{libexec}/git-*"]
```

The tools will still technically live in `libexec/"bin"` so they'll know where to find the helpers/drivers in the directory they expect.